source/Geometry/src/Layer.cc

```diff
@@ -24,12 +26,12 @@ Layer::Layer(int id, int barrelNumLayers, bool sameRods, bool barrelMinRFixed, b
  sameParityRods (       "sameParityRods"     , parsedAndChecked(), true),
  layerRotation  (       "layerRotation"      , parsedOnly()      , 0.),
  tiltedLayerSpecFile(   "tiltedLayerSpecFile", parsedOnly()),
- m_smallDelta   (       "smallDelta"         , parsedAndChecked()),
+ smallDelta     (       "smallDelta"         , parsedAndChecked()),
  m_smallParity  (       "smallParity"        , parsedAndChecked(),-1),
```

Comment:
Also here: I personally prefer smallParity to m_smallParity, but then if we want to be consistent, why not m_smallDelta?